stdlib/Statistics/src/Statistics.jl

```diff
@@ -74,6 +74,7 @@ function mean(f, itr)
     return total/count
 end
 mean(f, A::AbstractArray) = sum(f, A) / length(A)
+mean(f, A; dims = 1) = sum(f, A, dims = dims) / size(A, dims)
```

Comment:
The `dims` argument can be a tuple, in which case `/ size(A, dims)` won't work.

Reply:
Changed the implementation. Now, it should be fine!

Reply:
Add a test that would have caught this?

Reply:
Added.